examples/protyping/auto-segmentation/model_building.ipynb

```diff
@@ -9,7 +9,11 @@
   },
   {
    "cell_type": "code",
+<<<<<<< HEAD
+   "execution_count": 15,
+=======
    "execution_count": null,
+>>>>>>> 90184c9cc5017638aee4f6267356ad6324e88111
```

Comment:
oops :), this'll be a bit painful. These are previous merge conflicts that needed to be resolved...